src/components/InfoPanel.component.js

```diff
@@ -7,7 +7,19 @@ import './../styles/App.css'
 import './../styles/infopanel.css'
 import { utils } from './../utils'
 
+import { FieldGroup, LinkTable } from './'
+
 class InfoPanel extends Component {
+  constructor(props) {
+    super(props)
+    this.handleSelect = this.handleSelect.bind(this)
```

Comment:
if you use the arrow function syntax to define `handleSelect` (look at `onClickEdit`), you can get rid of this `constructor`